### DISCUSSION TITLE
OpenCV(4.6.0) :-1: error: (-5:Bad argument) in function 'cvtColor'

### DISCUSSION THREAD

**Discussion by Vinay1311** [datasciencecompleteguide/Machine-Learning#57] [Q&A]
from keras.models import load_model
import cv2
from keras.preprocessing import image
from keras_preprocessing.image import img_to_array
from time import sleep
face_classifier = cv2.CascadeClassifier(r'haarcascade_frontalface_default.xml')
classifier =load_model(r'best_model.h5')

emotion_labels = ['Angry','Disgust','Fear','Happy','Neutral', 'Sad', 'Surprise']

cap = cv2.VideoCapture(0)

while True:
    frame = cap.read()
    labels = []
    gray = cv2.cvtColor(frame, cv2.COLOR_BGR2RGB)
OpenCV(4.6.0) :-1: error: (-5:Bad argument) in function 'cvtColor'

Overload resolution failed:

src is not a numerical tuple
Expected Ptr<cv::UMat> for argument 'src'

*Getting above error while run that part of program
note: this is not a complete program.

**Comment by datasciencecompleteguide** [comment]
Does your camera open whenever you are running this program?
Because "Expected Ptrcv::UMat for argument 'src'" it's a bad argument error, i.e., either you are unable to provide argument to the funtion or its data type isn't correct!
So, Please Check for the same.
